pkgs/by-name/cl/cloudflare-utils/package.nix

```diff
@@ -0,0 +1,25 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "cloudflare-utils";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "Cyb3r-Jak3";
+    repo = pname;
```

Comment:
There's also a common argument against using `repo = pname` is that it creates a not really needed link between the two attributes.

Reply:
Updated